readme.md

```diff
@@ -570,6 +570,7 @@
 
 ## Higher Education
 
+- [Academic Writing](https://github.com/maehr/awesome-academic-writing) - Tools for distraction free academic writing with Markdown.
```

Comment:
PR Template discourages use of terms like "resources" and "tools" in the description as awesome lists are primarily tools and resources, so it is obvious. I would perhaps change it to something like:

```suggestion
- [Academic Writing](https://github.com/maehr/awesome-academic-writing) - Distraction-free academic writing with Markdown.
```